### PR TITLE
graalvm11-ce: 21.3.0 -> 22.0.0.2

### PR DESCRIPTION
**Pull request by ericdallo**
graalvm17-ce: 21.3.0 -> 22.0.0.2

###### Motivation for this change
Bump graalvm to latest

###### Things done
Ran update script

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/compilers/graalvm/community-edition/hashes.nix [hunk 1]
```diff
@@ -1,57 +1,57 @@
-# Generated by ./update.sh script
+# Generated by /home/greg/dev/nixpkgs/pkgs/development/compilers/graalvm/community-edition/update.sh script
```
Comment (thiagokokada) [inline pkgs/development/compilers/graalvm/community-edition/hashes.nix:1]: Please run this script from the proper location (inside the ` pkgs/development/compilers/graalvm/community-edition` directory).

Comment (thiagokokada) [reply pkgs/development/compilers/graalvm/community-edition/hashes.nix:1]: Or fix the script to use only the basename.

Comment (ericdallo) [reply pkgs/development/compilers/graalvm/community-edition/hashes.nix:1]: Done!

**Comment by thiagokokada** [comment]
@ofborg build babashka

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 156918` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>clojure-lsp</li>
    <li>vscode-extensions.betterthantomorrow.calva</li>
  </ul>
</details>
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>HentaiAtHome</li>
    <li>babashka</li>
    <li>clj-kondo</li>
    <li>dapl-native</li>
    <li>dbqn-native</li>
    <li>graalvm11-ce</li>
    <li>graalvm17-ce</li>
    <li>jet</li>
    <li>neil</li>
    <li>zprint</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
The `clojure-lsp` failure seems related to SVM, I think it is caused by Datalevin again? Since you're the maintainer @ericdallo, if you want to follow with this merge knowing this will break the `clojure-lsp` package (since I imagine you will need to fix this in `clojure-lsp`/`dataelvin` first), I can merge this.

`vscode-extensions.betterthantomorrow.calva` depends on `clojure-lsp` too, so once fixed it is fixed this also should be fixed.

**Comment by thiagokokada** [comment]
BTW, I want to see if this new version fixed `aarch64-linux` or introduced `aarch64-darwin`. But I will probably do this in a separate PR.

**Comment by ericdallo** [comment]
@thiagokokada no problem, I removed datalevin from clojure-lsp replaced by transit :) Next release will already avoid lots of graalvm issues, and I'll probably need to change the derivation so I'm ok with merging this

**Comment by thiagokokada** [comment]
> @thiagokokada no problem, I removed datalevin from clojure-lsp replaced by transit :) Next release will already avoid lots of graalvm issues, and I'll probably need to change the derivation so I'm ok with merging this

You may want to bump `clojure-lsp` first maybe them, since once datalevin is removed `clojure-lsp` should build with either version of GraalVM.

But your call. I am waiting just for the `x86_64-darwin` checks.

**Comment by thiagokokada** [comment]
Thanks @ericdallo.

**Pull request merged**